src/plugins/imagery/components/ImageryView.vue

```diff
@@ -87,13 +88,15 @@
             v-if="shouldDisplayCompass"
             :image="focusedImage"
             :sized-image-dimensions="sizedImageDimensions"
+            :show-compass-h-u-d="showCompassHUD"
           />
           <AnnotationsCanvas
             v-if="shouldDisplayAnnotations"
             :image="focusedImage"
             :imagery-annotations="imageryAnnotations[focusedImage.time]"
-            @annotationMarqueed="handlePauseButton(true)"
-            @annotationsChanged="loadAnnotations"
+            @annotationMarqueeStarted="pauseAndHideImageControls"
```

Comment:
Hmm. I'm a bit confused by the different event names. Can we change these to all be hyphenated or all be camel case?